src/views/events/Edit.vue

```diff
@@ -1,41 +1,9 @@
 <template>
   <div class="tile is-ancestor ">
     <div class="tile is-child">
-      <div v-if="$route.params.id">
-        <div class="subtitle">Update event image</div>
-
-        <div class="field is-grouped">
-          <div class="control">
-            <div class="file has-name">
-              <label class="file-label">
-                <input class="file-input" type="file" name="resume" @change="setFile($event)">
-                <span class="file-cta">
-                  <span class="file-icon">
-                    <font-awesome-icon icon="upload" />
-                  </span>
-                  <span class="file-label">
-                    Choose a file
-                  </span>
-                </span>
-                <span class="file-name">
-                  {{ file ? file.name : 'Not set.' }}
-                </span>
-              </label>
-            </div>
-          </div>
-
-          <div class="control">
-            <a class="button is-info" :disabled="!file" @click="updateImage()">Upload!</a>
-          </div>
-        </div>
-
-        <hr />
-      </div>
-
       <form @submit.prevent="saveEvent()">
         <div class="notification is-info" v-if="!$route.params.id">
           <div class="content">
-            <p>If you want to upload the image, please add it after creating the event by going to "Edit event" and uploading it there.</p>
```

Comment:
This just needs to be updated to point towards the profile, don't just remove it. Also add it in statutory and summeruniversity please